sklearn/svm/src/libsvm/svm.cpp

```diff
@@ -2878,9 +2910,10 @@ double PREFIX(predict_values)(const PREFIX(model) *model, const PREFIX(node) *x,
 		free(vote);
 		return model->label[vote_max_idx];
 	}
+	NAMESPACE::Kernel::free_array();
```

Comment:
This line will not be called if `return model->label...` is reached. Is this an issue here?

Reply:
@thomasjpfan Thanks for your careful review of my patch. I've fixed the issue and committed the code.